robosuite/wrappers/domain_randomization_wrapper.py

```diff
@@ -154,6 +155,10 @@ def __init__(
         self.modders = []
 
         if self.randomize_color:
+            assert mujoco.__version__ == "3.1.1", (
+                "TextureModder requires mujoco version 3.1.1 to run. "
+                "Pending support for later versions. Alternatively, you can set randomize_color=False."
+            )
```

Comment:
Let's use the robosuite logger function

Reply:
`assert robot is not None, "If controller is None, robot must be specified."`

In the repo, there are a lot of examples that use assertions without logging; is a particular reason to use logging here?

Reply:
I would suggest change those plain printing, instead of following them in the future. It's easier for people to spot the message if there is "[robosuite Error]" etc. People tend to miss those messages within irrelevant track-back messages.

Reply:
Gotcha, what kind of structure would you use for these logging assertions?

Reply:
something like:

````python
if [assert condition]:
   robosuite.logger("original assert message")
   assert False, "original assert message"?
```
?

Reply:
What kind of "structures" are you referring to? You mean like the format inside the brackets?

Reply:
I mean:

current structure: `assert condition, "message"`

new structure: ?

Reply:
Oh, sorry, I think I was under some impression.... How about having a function to format the message only? So that we can still have some colored prints. like assert condition, rs_assert_error("message").

Reply:
sure, one sec

Reply:
maybe just make it red or any good-looking color to highlight the message will be sufficient.

Reply:
Updated: now the assert error looks like
![image](https://github.com/user-attachments/assets/32e8b162-35fa-4160-9335-88f4a0670224)

Reply:
To get the same error message, set

`        randomize_color=True,  # randomize_color currently only works for mujoco==3.1.1`

in `python robosuite/demos/demo_domain_randomization.py`

Reply:
@zhuyifengzju does this look good to you?

Reply:
@zhuyifengzju does this look good to you?